e2e/scripts/import/common.sh

```
@@ -0,0 +1,252 @@
+}
+
+check_tools() {
+    if ! command -v jq &> /dev/null; then
+        echo "Error: jq is not installed. Please install it:"
+        echo "brew install jq"
+        exit 1
+    fi
+}
+
+build_jar() {
+    cd "$PROJECT_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)